outlook/indico_outlook/calendar.py

```diff
@@ -113,7 +115,11 @@ def _update_calendar_entry(entry, settings):
         logger.debug('User %s has disabled calendar entries', user)
         return True
 
-    unique_id = '{}{}_{}'.format(settings['id_prefix'], user.id, entry.event_id)
+    # Use common format for event calendar ID if the event was created on or after 2025-04-06
+    if entry.event.created_dt >= datetime(2025, 4, 6, tzinfo=pytz.UTC):
```

Comment:
Instead of using the date I would use a plugin setting for this that takes an event id, and then apply the new logic to events with an ID higher than that.

It's more reliable and avoids issues w/ events created between merging this with whatever hardcoded date and deploying it.

Reply:
Ok, done.